providers/frame/common.ts

```diff
@@ -58,17 +58,23 @@ export const createUsers ${
 }
 
 export async function handleCount(request: Request, cache: { count: number }): Promise<Response> {
-  const body = await request.json()
-  $.assert($.field("count", $.u32), body)
-  const { count } = body
-  const index = cache.count
-  const newCount = index + count
-  if (newCount < DEFAULT_TEST_USER_COUNT) cache.count = newCount
-  else throw new Error("Maximum test user count reached")
-  return new Response(JSON.stringify({ index }), {
-    status: 200,
-    headers: { "Content-Type": "application/json" },
-  })
+  switch (request.method) {
```

Comment:
Why was this change made?

Reply:
When it does cors it sends a OPTION request to this endpoint. that request doesnt have a body so it would just crash

Reply:
In that case, can you handle OPTIONS requests in `handleCors`?

Reply:
Wouldn't it still crash since this handler would trigger before `handleCors`?

Reply:
I guess it would be caught by the handleError we would just ignore the response

Reply:
> Wouldn't it still crash since this handler would trigger before `handleCors`?

I was suggesting that you check in `handleCors` if the method is `OPTIONS` before calling the inner handler